pyproject.toml

```diff
@@ -1,3 +1,102 @@
+[tool.poetry]
+name = "saleor"
+version = "2.8.0"
+description = "A modular, high performance e-commerce storefront built with GraphQL, Django, and ReactJS."
+authors = ["Mirumee Software <hello@mirumee.com>"]
+license = "BSD-3-Clause"
+
+readme = "README.md"
+
+homepage = "https://getsaleor.com/"
+repository = "https://github.com/mirumee/saleor"
+documentation = "https://docs.getsaleor.com/en/latest/"
+
+[tool.poetry.dependencies]
+python = "~3.7"
+babel = "~2.7"
+bleach = "~2"
+boto3 = "^1.9"
+braintree = "~3.49.0"
+celery = { version = "^4.3.0", extras = ["redis"] }
+dataclasses = { version = "^0.6", markers = "python_version < '3.7'" }
+dj-database-url = "^0"
+dj-email-url = "^0"
+django = "~2.2"
+django-babel = "^0"
+django-bootstrap4 = "^0"
+django-cache-url = "^3.0"
+django-countries = "^5.3"
+django-elasticsearch-dsl = "^0.5"
+django-filter = "^2.2"
+django-graphql-jwt = "^0"
+django-impersonate = "^1.4.1"
+django-measurement = "~3.0"
+django-mptt = "^0"
+django-phonenumber-field = "^2.4"
+django-prices = "^1.0.4"
+django-prices-openexchangerates = "^1.0.1"
+django-prices-vatlayer = "^1.0.2"
+django-recaptcha = "^1.5"
+django-redis = "^4.10"
+django-silk = "~2.0"
+django-storages = { version = "^1.7.1", extras = [ "google" ] }
+django-templated-email = "^2.3.0"
+django-versatileimagefield = "^1.10"
+django-webpack-loader = "^0"
+draftjs-sanitizer = "^1.0.0"
+elasticsearch = "~6.3.1"
+elasticsearch-dsl = "~6.0.1"
+faker = "^2.0"
+freezegun = "^0"
+google-cloud-storage = "^1.18.0"
```

Comment:
You might ask why I added this and the answer is because of some weird behaviour. It exports django-storages extras as `google-cloud-storage==1.18.0; extra = "google"`, then when it tried to install it (with pip, not poetry) it ignored those dependencies as `Ignoring google-cloud-storage: markers 'extra == "google"' don't match your environment`. This is the reason why CI failed, I have no idea why it works like that. For redis it seems to be just fine, that is it adds redis normally without any extra marker.